tests/Unit/PointwiseFunctions/Xcts/Test_AdmMomentum.cpp

```diff
@@ -32,15 +32,18 @@ using Schwarzschild = Xcts::Solutions::Schwarzschild;
 using KerrSchild = Xcts::Solutions::WrappedGr<gr::Solutions::KerrSchild>;
 
 template <typename Solution>
-void test_infinite_surface_integral(const double distance, const double mass,
-                                    const double horizon_radius,
-                                    const double boost_speed,
-                                    const Solution& solution) {
+void test_infinite_surface_integrals(const double distance, const double mass,
+                                     const double horizon_radius,
+                                     const double boost_speed,
+                                     const Solution& solution) {
+  // Define x-shift in the domain
+  const double x_shift = 1.;
+
   // Set up domain
   const size_t h_refinement = 1;
   const size_t p_refinement = 6;
   const domain::creators::Sphere shell{
-      /* inner_radius */ horizon_radius,
+      /* inner_radius */ horizon_radius + x_shift,
```

Comment:
This will only increase the inner radius, not shift coordinates

Reply:
This is increasing the domain inner radius so that we don't use an isotropic Schwarzschild solution inside the horizon. Line 99 below is responsible for shifting the coordinates.

Reply:
Ok I see now how you need this to evaluate the solution at the shifted coordinates.